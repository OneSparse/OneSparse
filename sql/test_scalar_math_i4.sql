set client_min_messages = 'WARNING';
create extension if not exists onesparse;
set search_path = 'onesparse';
select (-1::integer)::scalar + 1::integer;
select -1::integer + (1::integer)::scalar;
