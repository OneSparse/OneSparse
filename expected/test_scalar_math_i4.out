-- # Scalar `integer` Math
--
-- This documentation is also tests for the code, the examples below
-- show the literal output of these statements from Postgres.
--
-- Some setup to make sure warnings are shown, and that the extension
-- is installed.
set client_min_messages = 'WARNING';
create extension if not exists onesparse;
-- Add mixed scalar objects and postgres scalar values
select (-1::integer)::scalar + 1::integer;
 ?column? 
----------
 i4:0
(1 row)

select -1::integer + (1::integer)::scalar;
 ?column? 
----------
        0
(1 row)

